linux/files/etc_profile

```diff
@@ -0,0 +1,17 @@
+
+{%- for name,value in profiles.iteritems() %}
```

Comment:
Main purpose of profile.d is to place basically any script in there, not only for exporting env variables - this main purpose should be also main purpose of linux.system.profile